alpaca/options.go

```diff
@@ -0,0 +1,94 @@
+package alpaca
+
+import (
+	"cloud.google.com/go/civil"
+	"github.com/shopspring/decimal"
+)
+
+//go:generate go install github.com/mailru/easyjson/...@v0.7.7
+//go:generate easyjson -all -snake_case $GOFILE
```

Comment:
It's a bit strange that the "option entities" are now in `options.go` (not in `entities.go`), but the corresponding endpoint methods still in `rest.go`. I did it the other way around when adding options to the marketdata package: moving the entities to `entities.go` but having the methods in `options.go`. I'm fine with either of these solutions:
- no new file, add only to `entities.go` / `rest.go`
- all new options related stuff to `options.go`
- new options entities to `entities.go`, methods to `options.go`

Reply:
I'll go with the no new file.